.travis/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #  'RUN .travis/install-verilator.sh' to this dockerfile to have an image
 #  with the full environment already installed.
 #
-FROM ubuntu:trusty
+FROM librecores/librecores-ci:latest
```

Comment:
It we run librecores/librecores-ci image in our tests do we need this `.travis/Dockerfile` anymore?  Do we use this file?

Reply:
@stffrdhrn This dockerfile does the same work what it used to do before i.e to simulate the travis environment only for testing purposes,this is not actually used in travis . And it works fine Librecores-ci base image

Reply:
Yes, but we needed to simulate the Travis environment to have something consistent for running tests. Now the tests run in docker so it's consistent.  This is not needed. Anyway I can remove.